package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "predev": "rm -fr .next",
```

Comment:
Cleans the .next directory before starting the development server to ensure a fresh build environment.

    Prevents stale build cache issues
    Resolves unexpected behaviors from previous builds
    Ensures clean development environment on each start

Refs: https://x.com/jherr/status/1758571101964382487?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1758571101964382487%7Ctwgr%5Ef5886021905bc0d1bb0685696bd0e2f3b51a4328%7Ctwcon%5Es1_c10&ref_url=https%3A%2F%2Fembed.zenn.studio%2Ftweetzenn-embedded__c670a3c986375